pom.xml

```diff
@@ -121,6 +127,12 @@
             <artifactId>slf4j-api</artifactId>
         </dependency>
 
+        <!-- Utilities -->
+        <dependency>
+            <groupId>com.auth0</groupId>
+            <artifactId>java-jwt</artifactId>
+            <version>3.14.0</version>
```

Comment:
I guess it's worth putting this in the super pom at some stage soon so auth related things are running on a consistent version across components.